Command/ImportCommand.php

```diff
@@ -0,0 +1,174 @@
+    /** @var ContentTypeService */
+    private $contentTypeService;
+    /** @var DataService */
+    private $dataService;
+
+    public function __construct(Logger $logger, Client $client, ContentTypeService $contentTypeService, ImportService $importService, DataService $dataService)
+    {
+        $this->contentTypeService = $contentTypeService;
+        $this->importService = $importService;
+        $this->dataService = $dataService;
+        parent::__construct($logger, $client);
+    }
+    
+    protected function configure()
+    {
+        $this
+            ->setName('ems:document:import')
+            ->setDescription('Import a bunch of json files from a zip file')
+            ->addArgument(
+                'contentTypeName',
+                InputArgument::REQUIRED,
+                'Content type name to import into'
+            )
+            ->addArgument(
+                'archive',
+                InputArgument::REQUIRED,
+                'Zip file containing the json files'
+            )
+            ->addOption(
+                'bulkSize',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Size of the elasticsearch bulk request',
+                500
+            )
+            ->addOption(
+                'raw',
+                null,
+                InputOption::VALUE_NONE,
+                'The content will be imported as is. Without any field validation, data stripping or field protection'
+            )
+            ->addOption(
+                'dont-sign-data',
+                null,
+                InputOption::VALUE_NONE,
+                'The content will not be signed during the import process'
+            )
+            ->addOption(
+                'force',
+                null,
+                InputOption::VALUE_NONE,
+                'Also treat document in draft mode'
+            )
+            ->addOption(
+                'dont-finalize',
+                null,
+                InputOption::VALUE_NONE,
+                'Don\'t finalize document'
+            )
+            ->addOption(
+                'businessKey',
+                null,
+                InputOption::VALUE_NONE,
+                'Try to identify documents by their business keys'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $contentTypeName = $input->getArgument('contentTypeName');
+        $archiveFilename = $input->getArgument('archive');
+        $bulkSize = $input->getOption('bulkSize');
+        $rawImport = $input->getOption('raw');
+        $signData = !$input->getOption('dont-sign-data');
+        $finalize = !$input->getOption('dont-finalize');
+        $force = $input->getOption('force');
+        $replaceBusinessKey = $input->getOption('businessKey');
+
+
+        $contentType = $this->contentTypeService->getByName($contentTypeName);
+        if (!$contentType instanceof ContentType) {
```

Comment:
Execute should execute, not verify. Use the "interact" function for all verifications (idealy allow the user to correct in the interact fase). Cfr https://github.com/ems-project/EMSMakerBundle/blob/master/Command/ContentTypeCommand.php#L94

Reply:
a04dfe4f5733de32c981266d96292dc313ed9cb0